README.md

```diff
@@ -1,7 +1,77 @@
 # Grateful Together API
 
+## Abstract: 
+This is the Backend API for the [Grateful Together App](https://www.FIX-THIS.com). Grateful Together is an application that allows groups of people to keep a central, easy to use, daily gratitude journal.
 
-### JSON Contract
+### Preview of App:
+Live Deployment link
+[Frontend Repo](https://github.com/GratefulTogether/gratefultogether-ui) 
+
+Add GIF here
+
+### Context:
+Collaborative capstone fullstack project for Mod 4 of Turing School of Software and Design.
+
+### Contributors
+<table>
+  <tr>
+    <td> Kailey Kaes: <a href="https://github.com/kaileykaes">Github</td>
+    <td> Jesse Sorman: <a href="https://github.com/jesse193">Github</td>
+    <td> K.D. Hubbard: <a href="https://github.com/kdhubb">Github</td>
+    <td> Steven Stackhouse: <a href="https://github.com/stackmm">Github</td>
+  </tr>
+</table>
+
+### Learning Goals:
+
+Develop a fullstack application in collaboration with a frontend team. Specifically, the backend team was tasked with creating a RESTful API service for the Grateful Together application, implementing at least one new technology. Our team also learned and executed an agile process from planning to development, through production, as well as a CI/CD process using CircleCI. 
+
+New Technologies implemented:
+- ActionCable
+- WebSockets
+- Action-Cable-Testing
+- CI/CD with CircleCI
```

Comment:
I would move this all to the bottom on the repo. Typically usage and setup is one of the first things we want to see in a README.